README.md

```diff
@@ -1,39 +1,82 @@
+  <img src="public/images/UI-kit-header.png"/>
+  <h3 align="center">
+    <a href="https://propelify.propeldata.com" target="_blank">Demo Dashboard</a> |&nbsp; 
+    <a href="https://storybook.propeldata.com" target="_blank">Storybook</a>
+  </h3>
 </div>
 <br />
 
-## Introduction
+The Propel UI Kit is an open-source library of React components to quickly build awesome-looking dashboards and data visualizations.
+
+## Why the Propel UI Kit?
+
+Building data visualization UIs is never easy. Implementing designs from scratch is time-consuming, fetching data requires a lot of backend logic, and there are always a lot of edge cases to deal with, like the intricacies of the different time ranges, or handling the loading and error states.
+
+Charting libraries like ChartJS, Echarts, Highcharts, or D3 are great but still need a lot of work to build them into elegant and cohesive product experiences. On the flip side, embedding third-party chart widgets via iFrames into your customer-facing product results in an inflexible, slow, and inelegant product experience.
+
+That's why we built the Propel UI Kit. Its component-based approach allows you to move fast, look great, and focus your efforts on building components that are unique to your product.
+
+## Core principles
+
+The Propel UI Kit is an open-source project built on five core principles:
+
+- **Don't reinvent the wheel.** Charting libraries are great, let's make them better. We built the UI Kit on top of all the power and efficiency of ChartJS.
+
+- **Work and look great out-of-box.** The components work and look great out of the box. You can drop them into your app with minor color customizations and you'll be off to a great start.
+
+- **Extremely customizable.** You can customize their styles, look and feel, and error and loading experience to always feel native to your product.
 
-Propel UI Kit is a set of customer-facing components that are powered by Propel APIs. The aim of this project is to simplify the process of creating data visualizations for our customers, even for those who may not have much front-end development experience. This is particularly useful for teams that don't want to allocate resources to building user interfaces that will connect with Propel.
+- **Built in React for React.** We built them in React for React frontends. No cross-framework abstractions and no unnecessary baggage.
 
-Each component in the Propel UI Kit is connected to one metric and can automatically fetch that metric's data, making it easy for users to create visualizations without having to write custom code. Additionally, the components come with a variety of props that can be used to configure queries based on our [Propel GraphQL API](https://propeldata.com/docs/api/about-the-graphql-api). This enables users to customize the components to suit their specific needs.
+- **They work with or without Propel's APIs.** You can use them by themselves in static mode or with the Propel API in connected mode.
```

Comment:
Nit: we write "Static" and "Connected" below. Should be consistent everywhere, IMO.